infrastructure/monitoring.tf

```diff
@@ -0,0 +1,30 @@
+resource "azurerm_log_analytics_workspace" "main" {
+  name                = "${local.org}-log-${local.resource_suffix}"
+  location            = module.primary_region.location
+  resource_group_name = azurerm_resource_group.primary.name
+  sku                 = "PerGB2018"
+  retention_in_days   = 30
+
+  tags = local.tags
+}
+
+resource "azurerm_application_insights" "main" {
+  name                 = "${local.org}-ai-${local.resource_suffix}"
+  location             = module.primary_region.location
+  resource_group_name  = azurerm_resource_group.primary.name
+  workspace_id         = azurerm_log_analytics_workspace.main.id
+  application_type     = "other" # should this be Node.JS?
```

Comment:
Maybe `web`?
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/application_insights

Reply:
It says `web` is for `ASP.NET`, so I figured we either want general (other) or Node (Node.JS)?